java-components/build-request-processor/src/main/java/com/redhat/hacbs/container/analyser/deploy/DeployCommand.java

```diff
@@ -380,6 +393,13 @@ public void accept(String s, String hash) {
                     deploymentPath);
             deployer.deploy();
         }
+        if ((isNotEmpty(gitURL) || isNotEmpty(gitIdentity)) && gitToken.isPresent()) {
```

Comment:
I wonder if we want to do this first? I think it might be better to have a git branch with no corresponding deployment than a deployment with no source.

Reply:
Do you mean move it above image/maven repo deployment?

Reply:
Moved to https://github.com/redhat-appstudio/jvm-build-service/pull/895/files#diff-9b9ad1a14aa7ca87134f689dba59451f601be0ac2e865537bc68c543515bc827R153